### DISCUSSION TITLE
Anyone having problems with gallery after update?

### DISCUSSION THREAD

**Discussion by LastDAU** [datenstrom/community#785] [Ask a question] [answered]
Ups, after the update the Gallery does not work as before. 
Images / thumbnails are not reduced in size.

[gallery 22/0516/pic.*jpg - 250]

**Comment by schulle4u** [comment, marked as answer]
The [gallery extension](https://github.com/annaesvensson/yellow-gallery) has been updated to support different sorting methods (e.g. name, modified, size). The update procedure should have taken care of converting your shortcuts, however it is always wise to check manually afterwards. Sorry for the inconvenience. 🙂

Se also datenstrom/community#141

**Comment by LastDAU** [comment]
Thanks, but ;)  What should / can I do manually? I am concerned with the size of the thumbnails, not the sorting. I have more than 300 small galleries with different sizes. I am a bit lost at the moment ?! Please excuse the bad English, greetings from DE

**Reply by schulle4u** [reply]
Please update your installation, we've published a patch which hopefully should convert your content files. After running the update, please check the latest entries in your `system/extensions/yellow-website.log`, you should find something like "Apply patches for Datenstrom Yellow 0.8.20". Your shortcuts then should have the new format, e.g. old `[gallery 22/0516/pic.*jpg - 250]` converts to new `[gallery 22/0516/pic.*jpg name zoom 250]`. Let us know if you still have problems.

**Reply by LastDAU** [reply]
\o/ Great! It works, thank you :+1:

**Comment by annaesvensson** [comment]
I can confirm, after the patch there are no more problems.
